app/models/concerns/oregon_digital/document_metadata.rb

```diff
@@ -44,6 +43,11 @@ module DocumentMetadata
       property :table_of_contents, predicate: ::RDF::Vocab::DC.tableOfContents do |index|
         index.as :stored_searchable
       end
+      puts properties.keys
+
+      define_singleton_method :document_properties do
+        (properties.select { |_k, v| v.class_name.nil? }.keys - initial_properties)
```

Comment:
The assumption here is that any property with a `class_name` must be a controlled vocabulary.. it would be better if this checked for a class_name and that the class name doesn't include `ControlledVocabularies`?

Reply:
Thats a good question. The assumption was that the only properties with a class_name are the ones that are controlled. 
https://github.com/OregonDigital/OD2/blob/master/app/models/concerns/oregon_digital/generic_metadata.rb#L491 This is where the property gets a class_name associated with it. There SHOULD be no other properties that get a class_name associated with it. But we could inspect deeper and say `select all values with a classname that doesnt have ControlledVocabularies` so that if there are any other classes that get associated with a property, it wont skip those.